.testcaferc.json

```diff
@@ -3,9 +3,5 @@
     "chrome:headless"
   ],
   "concurrency": 2,
```

Comment:
change it to `1`, otherwise it opens two browsers locally and also default to `chrome` instead of headless ?

Reply:
You'll need to coordinate this change with DevEx's test team. This parallelism was added in #1188, so I'd prefer to leave this unchanged.

Reply:
That change didn't force how developers wanted to run testcafe through command line.. It has it's own command to run.. But here we are forcing every developer to change `testcaferc.js`, I think `testcaferc.js` should be optional to use when dev wants to run command line, NOT the default.